### PR TITLE
Changed default server for samples from IIS Express to Kestrel

### LINKED ISSUE #27630: Change default server for samples from IIS Express to Kestrel
Samples that use IIS Express in the repo don't work because they rely on a native component that hasn't been built. For example, MVC and Blazor samples don't work when run with IIS Express. There's no guidance we currently can give to users to get samples working with IIS Express.

Although we can invest time in addressing the build process for the samples so we ensure the native component is always built before the samples are run, a quick fix is to edit the `launchSettings.json` files for the samples so they all use Kestrel for the server.

### PR DESCRIPTION
**Pull request by ShreyasJejurkar**
Keeping kestrel as the first option, VS will automatically show as the default option selected for debugging! (If not, make sure you delete the `.vs` folder from the root of the cloned repo. )

I have kept some IIS servers test project `launchSettings.json` untouched because keeping IIS Express as the default option for them makes sense! 

Fixes https://github.com/dotnet/aspnetcore/issues/27630

### CODE REVIEW

**Comment by javiercn** [comment]
@MCCshreyas thanks for your contribution!

It looks great, the only comment I have is that I believe there might be missing projects that need this treatment, for example the ones in Blazor. Ideally we want to tackle all the projects in the repo in one go.

@Tratcher can you look at the ones in Server?

**Comment by MCCshreyas** [comment]
Sure @javiercn I will get those addressed in this PR only!

**Review by Tratcher — COMMENTED** [review]
The three projects under `/src/Servers/HttpSys/samples/` (except TestClient) should also have a launchsettings that only includes the Project profile, no IIS.

File: src/DefaultBuilder/testassets/CreateDefaultBuilderApp/Properties/launchSettings.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
+{
```
[the remark this answers is not in the document]
Comment (MCCshreyas) [reply src/DefaultBuilder/testassets/CreateDefaultBuilderApp/Properties/launchSettings.json:1]: I did not added any `launchSettings.json` explicitly! I will check this!

File: src/Servers/Kestrel/samples/Http3SampleApp/Properties/launchSettings.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+      },
+      "applicationUrl": "https://localhost:5001;http://localhost:5000"
+    },
+    "IIS Express": {
+      "commandName": "IISExpress",
+      "launchBrowser": true,
+      "environmentVariables": {
+        "ASPNETCORE_ENVIRONMENT": "Development"
+      }
+    }
```
[the remark this answers is not in the document]
Comment (MCCshreyas) [reply src/Servers/Kestrel/samples/Http3SampleApp/Properties/launchSettings.json:25]: OK, thanks. I will remove the IIS config directly!

Intension of PR was to give kestrel as default option, but as per your suggestion for this specific projects I will remove IIS config and keep kestrel only!

File: src/DefaultBuilder/testassets/CreateDefaultBuilderApp/Properties/launchSettings.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (MCCshreyas) [reply src/DefaultBuilder/testassets/CreateDefaultBuilderApp/Properties/launchSettings.json:1]: @Tratcher It was already there! I did not added that explicitly!

File: src/Servers/IIS/IIS/test/IIS.Tests/Properties/launchSettings.json [hunk 3]
```diff
@@ -0,0 +1,40 @@
+{
```
[the remark this answers is not in the document]
Comment (MCCshreyas) [reply src/Servers/IIS/IIS/test/IIS.Tests/Properties/launchSettings.json:1]: It was already there, but as per your suggestion, I deleted the `launchSettings.json`

File: src/Servers/testassets/ServerComparison.TestSites/Properties/launchSettings.json [hunk 4]
```diff
@@ -0,0 +1,40 @@
+{
```
[the remark this answers is not in the document]
Comment (MCCshreyas) [reply src/Servers/testassets/ServerComparison.TestSites/Properties/launchSettings.json:1]: It was there already. Now converted to project profile!

**Review by Tratcher — APPROVED** [review]

**Comment by Tratcher** [comment]
Thanks

**Pull request merged**